conjure-java-core/src/main/java/com/palantir/conjure/java/types/BeanBuilderGenerator.java

```diff
@@ -349,17 +367,36 @@ private CodeBlock typeAwareAssignment(EnrichedField enriched, Type type, boolean
                     ConjureCollections.class,
```

Comment:
I think we’re trying to keep the dependency footprint small so don’t want to necessarily pull in Guava, but we could simplify this with something like `Lists.newArrayList(Iterable)` / `Sets.newHashSet(Iterable)`